frontend/components/common/AuthPrompt.js

```diff
@@ -23,32 +23,36 @@ const TitleHeader = s.div`
   }
 `
 
-const AuthPrompt = () => (
+const AuthPrompt = ({ title, children, hasLogin = true }) => (
   <PhoneContainer>
     <Center>
       <TitleHeader>
         <Image src="/static/img/peoplelogo.png" />
-        <Title>One last step...</Title>
+        <Title>{title || 'One last step...'}</Title>
       </TitleHeader>
       <Margin>
         <Text>
-          To make the most of Penn Clubs features, like bookmarking and
-          subscribing to clubs, please login using your PennKey.
+          {children ||
+            'To make the most of Penn Clubs features, like bookmarking and subscribing to clubs, please login using your PennKey.'}
         </Text>
       </Margin>
-      <Margin>
-        <a
-          href={`${LOGIN_URL}?next=${
-            typeof window !== 'undefined' ? window.location.href : '/'
-          }`}
-          className="button is-link is-large"
-        >
-          <Icon alt="login" name="key" /> Continue to login
-        </a>
-      </Margin>
-      <SmallText>
-        <i>(We're sorry, we hate two-step too.)</i>
-      </SmallText>
+      {hasLogin && (
+        <>
+          <Margin>
+            <a
+              href={`${LOGIN_URL}?next=${
+                typeof window !== 'undefined' ? window.location.href : '/'
```

Comment:
```suggestion
               window ? window.location.href : '/'
```

might be simpler, not sure if it catches all the cases though.

Reply:
also might be able to make href=... one line if you make this change.

Reply:
This would actually result in an error instead of the intended behavior.

![image](https://user-images.githubusercontent.com/4441174/73112875-b810ec00-3ede-11ea-8cf1-7761abf0bd18.png)

Reply:
sad, kk